### PR TITLE
Restyle machine view

### PR DESCRIPTION
**Pull request by huwshimi**
Update the machine view to the new designs.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- open the machine view
- check that it matches the new designs: https://github.com/CanonicalLtd/blues_browser/issues/1041

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3653/
Test PASSed.

**Comment by hatched** [comment]
the S in Units is cut off
screenshot from 2015-11-23 13 55 59

**Comment by hatched** [comment]
+ sign is a little low... When services are properly placed the companion green checkmark however appears to be in the proper spot

screenshot from 2015-11-23 13 58 40

**Comment by hatched** [comment]
:+1: QA :ok: with trivials

Thanks for doing these updates it looks great!

**Comment by huwshimi** [comment]
Thanks @hatched I've fixed those issues.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3662/
Test PASSed.

**Comment by kadams54** [comment]
QA issues:

* `.machine-token .token.active` seemed to be picking up incorrect styles; there was a negative margin-right and a solid right border, which resulted in this: active machine token
* We need `max-width` and `text-overflow` set on the scale up UI; when the name expands too far, all hell breaks loose: scale up overflow
* The unplaced units column needs to allow overflow for the dropdown menu on unplaced units: unplaced dropdown

File: jujugui/static/gui/src/app/assets/css/machine-view/_machine-token.scss [hunk 1]
```diff
@@ -1,5 +1,16 @@
 .machine-token {
     .token {
+        &.active {
+            margin-right: -21px;
+            padding-right: 31px;
+            border-style: dashed;
+            border-right: 1px solid $canvas-background;
+
```
Comment (kadams54) [inline jujugui/static/gui/src/app/assets/css/machine-view/_machine-token.scss]: OK, so this actually caused one of my QA issues. I went back to the mocks, scratched my head, and said… "Oh, it's really supposed to be like that. Hmm. That's interesting." I think we should raise it with UX; it looks (to me) like a CSS bug, not intentional design.

**Comment by huwshimi** [comment]
Thanks @kadams54, I've fixed the second two issues, the first issue needs some input from UX as discussed in IRC.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3668/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3669/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3670/
Test PASSed.

**Comment by hatched** [comment]
Thanks for updating these styles @huwshimi :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**